### ISSUE TITLE
IXF-Importer: reorder of email content

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/peeringdb#878]
Put the line 

```
You may review and manually accept the IX-F data at https://peeringdb.com//cp/peeringdb_server/ixfmemberdata/nnnn/change/
```

at the bottom of the email. When quoting that would allow removing internal information with a single action.

@peeringdb/pc, please +1

**Comment by shane-kerr** [comment]
+1

**Comment by ynbrthr** [comment]
+1

**Comment by peterhelmenstine** [comment]
+1

**Comment by ccaputo** [comment]
With upcoming #875 change, in which the same content goes to PDB admins & ix/network, I recommend the line be moved to the bottom and changed to:

```
PeeringDB Admins Only: http://localhost/cp/peeringdb_server/ixfmemberdata/nnn/change/
```

**Comment by egfrank** [comment]
** Summary **
- Have link at the bottom of IXF emails that states
```
PeeringDB Admins Only: http://localhost/cp/peeringdb_server/ixfmemberdata/nnn/change/
```

**Comment by arnoldnipper** [comment]
Only publish the URL if `ixf_ixp_member_list_url_visible` = "Public", @egfrank

**Comment by ccaputo** [comment]
> Only publish the URL if `ixf_ixp_member_list_url_visible` = "Public", @egfrank

@arnoldnipper, great point.  @egfrank, copying from https://github.com/peeringdb/peeringdb/issues/875#issuecomment-731408103 : 

> Additional aspect to be included: Based on "IX-F Member Export URL Visibility" the Deskpro ticket created should only include the IX-F URL if visibility is set to "Public", since the email may go to parties other than the IX. When public:
> 
> ```
> - IX-F Data: [URL]
> ```
> 
> When not public:
> 
> ```
> - IX-F Data: Not public
> ```

**Comment by ccaputo** [comment]
> Only publish the URL if `ixf_ixp_member_list_url_visible` = "Public", @egfrank

I deleted my previous comment after re-remembering these are two different things.  @arnoldnipper, this is different than https://github.com/peeringdb/peeringdb/issues/875#issuecomment-731408103 which references the the IX-F JSON URL.  Rather, here the reference is to an internal PeeringDB URL, not the IX-F JSON URL, thus I believe @egfrank's Summary is correct.

**Comment by arnoldnipper** [comment]
> I deleted my previous comment after re-remembering these are two different things. @arnoldnipper, this is different than [#875 (comment)](https://github.com/peeringdb/peeringdb/issues/875#issuecomment-731408103) which references the the IX-F JSON URL. Rather, here the reference is to an internal PeeringDB URL, not the IX-F JSON URL, thus I believe @egfrank's Summary is correct.

I know that these are two different things. Though this email is internal, @peeringdb/ac quotes it to send a message to the network. From an @peeringdb/ac pov you could even drop this line. The reason I want to have it at the bottom is to be able to easily remove it when quoting.

**Comment by ccaputo** [comment]
> > I deleted my previous comment after re-remembering these are two different things. @arnoldnipper, this is different than [#875 (comment)](https://github.com/peeringdb/peeringdb/issues/875#issuecomment-731408103) which references the the IX-F JSON URL. Rather, here the reference is to an internal PeeringDB URL, not the IX-F JSON URL, thus I believe @egfrank's Summary is correct.
> 
> I know that these are two different things. Though this email is internal, @peeringdb/ac quotes it to send a message to the network. From an @peeringdb/ac pov you could even drop this line. The reason I want to have it at the bottom is to be able to easily remove it when quoting. So, if you move the text to the 2nd last line and obey visibility we could even leave it in.

The email is not necessarily internal.  Once #875 is implemented, the Importer generated Deskpro tickets may CC: external contacts when able.

The "https://peeringdb.com//cp/peeringdb_server/ixfmemberdata/" URL is _only_ present for use by the @peeringdb/ac, as a convenient link should an admin decide to make the suggested change.  Users can not use it.  Are you saying you don't want it to be included anymore?

**Comment by arnoldnipper** [comment]
> he email is not necessarily internal. Once #875 is implemented, the Importer generated Deskpro tickets may CC: external contacts when able.
> 
> The "https://peeringdb.com//cp/peeringdb_server/ixfmemberdata/" URL is _only_ present for use by the @peeringdb/ac, as a convenient link should an admin decide to make the suggested change. Users can not use it. Are you saying you don't want it to be included anymore?

Where did I say that? My remarks are only about the IX-F JSON URL.

**Comment by ccaputo** [comment]
> > he email is not necessarily internal. Once #875 is implemented, the Importer generated Deskpro tickets may CC: external contacts when able.
> > The "https://peeringdb.com//cp/peeringdb_server/ixfmemberdata/" URL is _only_ present for use by the @peeringdb/ac, as a convenient link should an admin decide to make the suggested change. Users can not use it. Are you saying you don't want it to be included anymore?
> 
> Where did I say that? My remarks are only about the IX-F JSON URL.

This ticket was not originally about the IX-F JSON URL, it is about the internal /ixfmemberdata/ URL.  Thus when you wrote:

> From an @peeringdb/ac pov you could even drop this line.

I took that to be about the line indicated in the first message of this issue:

```
You may review and manually accept the IX-F data at https://peeringdb.com//cp/peeringdb_server/ixfmemberdata/nnnn/change/
```

Upon closer reading I see:

> So, if you move the text to the 2nd last line and obey visibility we could even leave it in.

and I suppose you mean the IX-F JSON URL line in that context.  It would be good to be explicit about the lines you are talking about.

Also, to be very clear, neither of these lines are present in the non-Deskpro emails the Importer sends out.  It is not the intent of this issue to add lines to the non-Deskpro emails the Importer sends out.  Rather these lines are only in the Deskpro tickets which are created, and which because of #875, will soon CC: networks/IXes upon creation, in addition to being seen by PeeringDB admins.

Thus I am guessing what you mean to propose is:

**Summary**:

For Deskpro ticket creation (and resulting emails), rearrange lines and factor in "IX-F Member Export URL Visibility" such that when that setting is `public`:

```
- IX-F Data: [URL]
- PeeringDB Admins Only: http://peeringdb.com/cp/peeringdb_server/ixfmemberdata/nnn/change/
```

When not `public`:

```
- IX-F Data: Not public
- PeeringDB Admins Only: http://peeringdb.com/cp/peeringdb_server/ixfmemberdata/nnn/change/
```

**Comment by arnoldnipper** [comment]
**Release Notes**
This is mainly an internal improvement. And fixes also not disclosing private IXP information to the public.

**Comment by vegu** [comment]
this was fixed in #875 - closing
